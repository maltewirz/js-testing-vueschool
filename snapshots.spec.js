const { TestScheduler } = require("jest")

const user = {
    name: 'El Franco',
    age: 42,
    job: 'Investor'
}

test('user matches', () => {
    // const userString = '{"name":"El Franco","age":42,"job":"Investor"}';
    // expect(JSON.stringify(user)).toEqual(userString);
    expect(user).toMatchSnapshot()
})

