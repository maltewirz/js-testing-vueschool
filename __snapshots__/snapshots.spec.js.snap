// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
Object {
  "age": 42,
  "job": "Investor",
  "name": "El Franco",
}
`;
